_posts/2017-08-25-introducing-sandboxfs.md

```diff
@@ -3,40 +3,46 @@ layout: posts
 title: Introducing sandboxfs
 ---
 
+sandboxfs is a new sandboxing technique for Bazel. It's experimental and
+subject to change, but it's available now for you to check out!
+
+# Correct builds
+
 As our motto claims, *correctness* is an integral part of Bazel. To achieve
 correctness, builds must be *hermetic* and
 [*reproducible*](https://reproducible-builds.org/), which means that all
 actions invoked by Bazel should be run in a well-defined environment. In other
 words: we want actions to run within a sandbox.
```

Comment:
This sounds like sandboxing is the *only* way to do this (but it's not - a system could also use containers, right?). Is sandboxing the only way to do this for Bazel, or is it the most realistic way to do it for Bazel?

Reply:
The way I see it, sandboxing is a general term and containers are a specific way of sandboxing invocations.

Reply:
Sounds good. Just want to ensure we're making the 1:1 comparison on purpose.